src/style-config/buttons.scss

```diff
@@ -28,6 +28,7 @@ values are [ background/border color, hover, active, text ]
 $buttonColorSets: (
  ("blue", $blue, $azul, $blue-bright, $white),
  ("magenta", $magenta, $magenta-dull, $magenta-light, $white),
+ ("magenta-dull", $magenta-dull, mix(black, $magenta-dull, 20), mix(white, $magenta-dull, 20), $white),
```

Comment:
I'm not blocking on this, but should we repeat this pattern for all the colours here? I feel like this could be an option, but also that the black / white mix might need to be different depending on the shade @harpalsingh curious